itou/templates/apply/list_for_siae.html

```diff
@@ -9,6 +10,21 @@ <h1>{% trans "Candidatures reçues" %}</h1>
 
     <h2 class="text-muted">{{ siae.display_name }}</h2>
 
+    <form method="get" action="">
```

Comment:
J'allais te suggérer un `{% csrf_token %}` pour la sécu comme on le fait déjà sur nos 28 (!) autres forms mais après lecture de https://security.stackexchange.com/questions/115794/should-i-use-csrf-protection-for-get-requests manifestement ce n'est pas forcément pertinent pour les forms en GET.

Bref ma sugesstion est overkill :-D

Reply:
:-D
J'ai eu la même réflexion que toi :)